app/views/steps/submission/shared/_percentage_answer.html.erb

```diff
@@ -0,0 +1,12 @@
+<%
+  content_for(:row_question, flush: true) do
+    t("summary.questions.#{percentage_answer.question}.question")
+  end
+
+  content_for(:row_answer, flush: true) do
+    number_to_percentage(percentage_answer.value, precision: 0) if percentage_answer.value
```

Comment:
Do we know for certain that there won't be decimal percentages? This obviously looks neater, but I wonder if people can own 50.5% of a house 🤷‍♂️  - could be worth double checking with BA